tasks/create-advisory/create-advisory.yaml

```diff
@@ -55,10 +55,10 @@ spec:
         RESULTS_FILE="$(workspaces.data.path)/$(params.resultsDirPath)/create-advisory-results.json"
 
         # Obtain application from snapshot
-        application=$(jq -rc .application "$(workspaces.data.path)/$(params.snapshotPath)")
+        application=$(jq -r .application "$(workspaces.data.path)/$(params.snapshotPath)")
 
         # Obtain origin workspace from releasePlanAdmission
-        origin=$(jq -rc '.spec.origin' "$(workspaces.data.path)/$(params.releasePlanAdmissionPath)")
+        origin=$(jq -r '.spec.origin' "$(workspaces.data.path)/$(params.releasePlanAdmissionPath)")
```

Comment:
Hmm, this is a string, right? So yeah, I agree. But how did this work before? There must have been redundant `""` in the string passed to the IR...

Reply:
My understanding is -rc on a string works just fine, but the -c doesn't really do anything